sos/report/plugins/azure.py

```diff
@@ -66,4 +67,21 @@ def setup(self):
                 'openssl x509 -noout -text -in ' + crt_path
             ])
 
+
+class AzureEnhancedMonitoring(Azure, RedHatPlugin):
+
+    def setup(self):
+        super(AzureEnhancedMonitoring, self).setup()
+        aem_settings = '/var/lib/waagent/' \
+            'Microsoft.OSTCExtensions.AzureEnhancedMonitorForLinux-*/' \
+            'config/0.settings'
+        # Check if AEM (Azure Enhanced Monitoring for Linux) is active
+        aem_glob = glob.glob(aem_settings)
+        if aem_glob:
+            if os.path.isfile(aem_glob[0]):
+                # Add AEM PerfCounters
+                self.add_copy_spec([
+                    '/var/lib/AzureEnhancedMonitor/PerfCounters'
+                ])
```

Comment:
No need for a list here - `add_copy_spec()` accepts either a list or a single string value as its first argument.

Reply:
OK. I wasn't aware of that and will adapt, once we resolve the discussion around the overall glob-ing.